.github/workflows/integration.yaml

```diff
@@ -107,7 +107,17 @@ jobs:
           TEST_VERSION_UPGRADE_MIN_RELEASE: "1.31"
           TEST_MIRROR_LIST: '[{"name": "ghcr.io", "port": 5000, "remote": "https://ghcr.io", "username": "${{ github.actor }}", "password": "${{ secrets.GITHUB_TOKEN }}"}, {"name": "docker.io", "port": 5001, "remote": "https://registry-1.docker.io", "username": "", "password": ""}, {"name": "rocks.canonical.com", "port": 5002, "remote": "https://rocks.canonical.com/cdk"}]'
         run: |
-          cd tests/integration && sg lxd -c 'tox -e integration -- --tags pull_request'
+          tags="pull_request"
+          # Run all tests if there are test changes. In case of a PR, we'll
+          # get a merge commit that includes all changes.
+          if git diff HEAD HEAD~1 --name-only | grep "tests/"; then
+            tags="weekly"
```

Comment:
this needs to be `up_to_weekly` otherwise only the weekly tags are executed. Same below.
See https://github.com/canonical/k8s-snap/blob/aabe445a8f41689f5dad577e1ac1e7091778ad13/tests/integration/tests/test_util/tags.py#L13

Reply:
Good point, thanks!